mvc/controllers/dependency-injection.rst

```diff
@@ -1,9 +1,6 @@
 Dependency Injection and Controllers
 ====================================
 
-.. _dependency-injection-controllers:
```

Comment:
To fix the Travis build. The change was already made upstream. It will go away from my diff when I rebase